app/views/idv/in_person/address.html.erb

```diff
@@ -103,4 +124,4 @@
   <%= render 'idv/doc_auth/cancel', step: 'address' %>
 <% end %>
 
-<%= javascript_packs_tag_once('formatted-fields') %>
+<%= javascript_packs_tag_once('formatted-fields', 'puerto-rico-hint') %>
```

Comment:
do we want to keep this in this pr or save for later?

Reply:
Puerto Rico hint text changes removed. There is another story for this change.